Tools/Scripts/generate-compile-commands

```diff
@@ -106,14 +108,18 @@ except Exception:
 
 # Write the new compile_commands.json file
 new_compile_commands_file = open("compile_commands.json", 'w')
-new_compile_commands_file.write("[\n")
+new_compile_commands_file.write("[")
 
 # We should probably remove the trailing comma in the last entry, but
```

Comment:
One more thing, we can probably remove this comment then.

Reply:
Sounds good.